bucket/strokesplus.net.json

```diff
@@ -0,0 +1,39 @@
+{
+    "version": "0.5.6.6",
+    "description": "Mouse gesture recognition utility for Windows which allows you to create powerful mouse gestures that save you time.",
+    "homepage": "https://www.strokesplus.net",
+    "license": {
+        "identifier": "Freeware",
+        "url": "https://www.strokesplus.net/FAQ"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://cdn.strokesplus.net/files/StrokesPlus.net_Portable_0.5.6.6.zip",
+            "hash": "b1c82604910676bc7bb3cd43b02a278bc01f51b212a6c180aab59f0a6ec86e9f"
+        },
+        "32bit": {
+            "url": "https://cdn.strokesplus.net/files/StrokesPlus.net_Portable_0.5.6.6.zip",
+            "hash": "b1c82604910676bc7bb3cd43b02a278bc01f51b212a6c180aab59f0a6ec86e9f"
+        }
+    },
+    "shortcuts": [
+        [
+            "StrokesPlus.net.exe",
+            "StrokesPlus.net"
+        ]
+    ],
+    "checkver": {
+        "url": "https://www.strokesplus.net/ChangeLog.txt",
+        "regex": "([\\d.]+)"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://cdn.strokesplus.net/files/StrokesPlus.net_Portable_$version.zip"
+            },
+            "32bit": {
+                "url": "https://cdn.strokesplus.net/files/StrokesPlus.net_Portable_$version.zip"
+            }

```

Comment:
Please also remove the architecture here.

Reply:
Ok.. you are suggesting that it should look like this: 
```
"64bit": {
             "url": "the_url_line" 
}
```
both "architecture" and "32bit" are removed. is that correct?

Reply:
> both "architecture" and "32bit" are removed. is that correct?

No. It should be something like:

```
"url": "https://cdn.strokesplus.net/files/StrokesPlus.net_Portable_0.5.6.6.zip",
"hash": "b1c82604910676bc7bb3cd43b02a278bc01f51b212a6c180aab59f0a6ec86e9f"

....

"autoupdate": {
    "url": "https://cdn.strokesplus.net/files/StrokesPlus.net_Portable_$version.zip"
}
```

Reply:
ah got it.